src/lib/transaction_snark/transaction_snark.ml

```diff
@@ -136,6 +136,8 @@ module Statement = struct
               , 'pending_coinbase
               , 'local_state )
               Registers.Stable.V1.t
+          ; connecting_ledger_left : 'ledger_hash
```

Comment:
These values need to be "nullable". Can these be options? Or is it easier (for inside the snark) to treat a sigil values such as a hash of `0` to be the "null" value?

Reply:
I suppose we could use zero. Can it not be the target ledger hash? For payments, I used target hash of first pass ledger for second pass ledgers and connecting ledgers